packages/types/src/hatchet.ts

```
@@ -0,0 +1,91 @@
+  redisCache?: Redis
+  prisma: PrismaClient
+}
+
+// Shared contract for Hatchet task handler injections.
+export interface HatchetHandlers {
+  handleSendTeamsNotifications: (
+    { scope, text }: { scope: string; text: string },
+    ctx: HatchetHandlerContext
+  ) => Promise<unknown> | void
+  handleUpdateGroupAverageScores: (
+    {},
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)